ethpm_types/contract_type.py

```diff
@@ -46,6 +46,74 @@ class ContractInstance(BaseModel):
     runtime_bytecode: Optional[Bytecode] = Field(None, alias="runtimeBytecode")
 
 
+class SourceMapItem(BaseModel):
+    # NOTE: `None` entry means this path was inserted by the compiler during codegen
+    start: Optional[int]
+    stop: Optional[int]
+    contract_id: Optional[int]
```

Comment:
We explicitly set `Optional[...] = None` on the other Pydantic models, should we take that approach here as well?
```suggestion
    start: Optional[int] = None
    stop: Optional[int] = None
    contract_id: Optional[int] = None
```

Reply:
So those are default values, for when you don't specify it when creating the model. E.g.:
```py
a = SourceMapItem()
b = SourceMapItem(1)  # same as start=1
...
```

However, we're always explicitly setting all items, so it's not really necessary here